src/Middleware/RoleMiddleware.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Spatie\Permission\Middleware;
+
+use Auth;
+use Closure;
+
+class RoleMiddleware
+{
+    public function handle($request, Closure $next, $role)
+    {
+        if (Auth::guest()) {
+            return $next($request);
```

Comment:
This doesn't seem right. It looks like guest can now access protected page. Could you add a test that makes sure that guest cannot access protected pages and fix this?